content/creator/scenes/releases/version-support-agreement.md

```diff
@@ -56,6 +56,10 @@ Once a major release reaches a certain level of maturity, a **beta** release can
 
 Once the development of the version family is considered to be ready for production, a **GA** (general availability) release can be made available. From this point onwards, the version is considered the recommended and encouraged option for all developers to use.
 
+### Stable releases with unstable namespaces
+
+In exceptional cases, a stable (beta or GA) version may include specific features grouped into a legacy namespace that is not considered to be part of the supported release and is meant to be deprecated in the future. This namespace will be labeled as unstable, and the documentation will make it clear when this is the case. The purpose of leaving these legacy features is to aid in the transition between major versions, without losing any functionality. This can happen when a proper re-implementation of certain features is required, but it would delay releasing the stabe version too much. Creators who make use of these legacy features must be aware that the feature will be replaced in future versions. In these cases, the changes will be communicated clearly, including migration guides, and there will be a transition period for creators to adjust their scene's code.
```

Comment:
Typo in "but it would delay releasing the stabe version too much" -- should be "stable".